code/game/gamemodes/changeling/powers/absorb.dm

```diff
@@ -2,7 +2,7 @@
 	name = "Absorb DNA"
 	desc = "Absorb the DNA of our victim."
 	chemical_cost = 0
-	dna_cost = 0
+	dna_cost = -1

```

Comment:
this is shit.

You've set almost every (if not all) stings to -1, to disable them.
If you are going to remove them, then actually put in the effort to remove them.